tests/test_table.py

```diff
@@ -260,7 +262,7 @@ def test_update_invalid_fields(TestModel, TestModel_entries, dynamo_local):
             unknown_attr='foo'
         )
 
-    with pytest.raises(InvalidSchemaField):
+    with pytest.raises(ConditionFailed):
```

Comment:
I would like to have kept this as an `InvalidSchemaField` from a semantic view point, but now that `Q` is a standalone thing that's not part of a table object it doesn't have access to a schema anymore and technically the condition is failing -- so I'm OK with the change here.